src/fsw/FCCode/Drivers/Piksi.cpp

```diff
@@ -3,6 +3,9 @@
 
 #ifndef DESKTOP
 #include <Arduino.h>
+// The below are declared in Arduino.h
+static void interrupts() {}
+void noInterrupts() {}
 #endif

```

Comment:
Shouldn't this be something like:
```suggestion
#ifndef DESKTOP
#include <Arduino.h>
#else
// The below are declared in Arduino.h
static void interrupts() {}
static void noInterrupts() {}
#endif
```